roles/openshift_health_checker/openshift_checks/docker_image_availability.py

```diff
@@ -0,0 +1,137 @@
+# pylint: disable=missing-docstring
+from openshift_checks import OpenShiftCheck, get_var
+
+
+class DockerImageAvailability(OpenShiftCheck):
+    """Check that required Docker images are available.
+
+    This check attempts to pull required docker images and fail if
+    any image is missing or an error occurs during pull.
+    """
+
+    name = "docker_image_availability"
+    tags = ["preflight"]
+
+    skopeo_image = "juanvallejo/skopeo:latest"
```

Comment:
Adding `:latest` is a no-op in this case, since that is the default tag. It doesn't make `docker run` re-pull the image -- but I confess there's a long time I don't read those docs, I might be wrong.
I think there might be a flag to force pull? Can't remember.